downstream/modules/hub/proc-using-podman-ensure-image-signed.adoc

```diff
@@ -5,9 +5,9 @@ When ensuring a signature is signed by specific signatures, the signature must b
 
 .Procedure
 
-. From the navigation panel, select menu:Signature Keys[].
+. From the navigation panel, select {MenuACAdminSignatureKeys}.
```

Comment:
This one is showing as menu: Signature Keys.

Reply:
Fixed.